### DISCUSSION TITLE
Dependancies

### DISCUSSION THREAD

**Discussion by sophieastrof** [GuyTeichman/RNAlysis#3] [Q&A] [answered]
Hi ! I managed to install Graphviz, R, Kallisto, and RNAlysis and can open RNAlysis GUI. But 
How do I automatically install the list of dependencies you listed? 
Thanks,
Sophie

**Comment by GuyTeichman** [comment, marked as answer]
Hi Sophie,
I apologise for the very late response, I must have missed the notification for your question. 

Other than Graphviz, R, and _kallisto_, all other dependencies should be installed automatically when you install _RNAlysis_. 
If you used the command `pip install RNAlysis`, then only the basic dependencies of _RNAlysis_ will be installed (likely to work out of the box on any computer). 
To install all of the dependencies (which may not work on every setup), use the command `pip install RNAlysis[all]`. 
This command should work even if you already have a version of _RNAlysis_ installed. 
If you also want to make sure you are running the latest version, use the command `pip install --upgrade RNAlysis[all]` - this will both install all dependencies, and upgrade _RNAlysis_ to the latest version. 

Feel free to comment or create a new post if you have any questions or run into trouble when installing the dependencies. 

Good luck,
Guy.
